contrib/fuzz/metadata_fuzz_test.go

```diff
@@ -49,14 +49,29 @@ func testEnv(t *testing.T) (context.Context, *bolt.DB, func(), error) {
 	}, nil
 }
 
+const (
+	// common operations
+	opCreate = "Create"
```

Comment:
Can you move these into the functions where they are used?  They can be kept in sync there, but we don't need to have them abstracted across functions.

https://go.dev/play/p/R8yIFa9LNF9

Reply:
ah SGTM. updated